src/theme/components/button.modifier.ts

```diff
@@ -12,6 +12,7 @@ export const MuiButton: Components<Theme>['MuiButton'] = {
       } = theme;
       return {
         ...textB2SemiBold,
```

Comment:
@aabidsofi19 this textB2SemiBold already passing the font family according to figma documentation